pkg/util/testing/wrapper.go

```diff
@@ -171,6 +230,50 @@ func (j *JobSetWrapper) ContainerTrainerEnv(env []corev1.EnvVar) *JobSetWrapper
 	return j
 }
 
+func (j *JobSetWrapper) Volumes(rJobName string, v ...corev1.Volume) *JobSetWrapper {
+	for i, rJob := range j.Spec.ReplicatedJobs {
+		if rJob.Name == rJobName {
+			j.Spec.ReplicatedJobs[i].Template.Spec.Template.Spec.Volumes = append(
+				j.Spec.ReplicatedJobs[i].Template.Spec.Template.Spec.Volumes,
+				v...,
+			)
+		}
+	}
+	return j
+}
+
+func (j *JobSetWrapper) VolumeMounts(rJobName, containerName string, vms ...corev1.VolumeMount) *JobSetWrapper {
+	for i, rJob := range j.Spec.ReplicatedJobs {
+		if rJob.Name == rJobName {
+			for k, container := range j.Spec.ReplicatedJobs[i].Template.Spec.Template.Spec.Containers {
+				if container.Name == containerName {
+					j.Spec.ReplicatedJobs[i].Template.Spec.Template.Spec.Containers[k].VolumeMounts = append(
+						j.Spec.ReplicatedJobs[i].Template.Spec.Template.Spec.Containers[k].VolumeMounts,
+						vms...,
+					)
+				}
+			}
+		}
+	}
+	return j
+}
+
+func (j *JobSetWrapper) Env(rJobName, containerName string, envs ...corev1.EnvVar) *JobSetWrapper {
```

Comment:
Please can you add TODO statement in the code that these functions should be refactored in favour of this Env() API:
- https://github.com/kubeflow/trainer/blob/76dff03117d3a191b2c625b116344f980d389686/pkg/util/testing/wrapper.go#L220
- https://github.com/kubeflow/trainer/blob/76dff03117d3a191b2c625b116344f980d389686/pkg/util/testing/wrapper.go#L293
- https://github.com/kubeflow/trainer/blob/76dff03117d3a191b2c625b116344f980d389686/pkg/util/testing/wrapper.go#L319

Reply:
Sure.

Reply:
Done.